luaa.c

```diff
@@ -612,7 +613,9 @@ setup_awesome_signals(lua_State *L)
 
     /* POSIX.1-2001, according to man 7 signal */
     SETUP_SIGNAL(SIGBUS);
+#ifndef __FreeBSD__
     SETUP_SIGNAL(SIGPOLL);
+#endif
```

Comment:
Shouldn't this check for if `SIGPOLL` is defined?!  (and then leave a comment about FreeBSD there only)